pulpcore/pulpcore/app/models/repository.py

```diff
@@ -247,6 +249,62 @@ def content(self):
         )
         return Content.objects.filter(version_memberships__in=relationships)
 
+    @property
```

Comment:
These 3 properties are probably temporary, pending a better solution.  This is ugly but it was fairly simple for an initial implementation.